src/utils/arrays.ts

```diff
@@ -223,6 +223,21 @@ export function arrayMerge<T>(...a: T[][]): T[] {
     }, new Set<T>()));
 }
 
+/**
+ * Moves a single element from fromIndex to toIndex.
+ * @param list the list from which to construct the new list.
+ * @param fromIndex the index of the element to move.
+ * @param toIndex the index of where to put the element.
+ * @returns A new array with the requested value moved.
```

Comment:
types

Reply:
Do we really need to duplicate types between TS signatures and JSDoc?

Reply:
our style guide says we do, yes.